portfolio/src/main/webapp/comments.html

```diff
@@ -1,37 +1,39 @@
 <!DOCTYPE html>
-    <html>
-      <head>
-        <meta charset="UTF-8">
-        <title>Jordan's Site</title>
-        <link rel="stylesheet" href="style.css">
-        <script src="script.js"></script>
-      </head>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Jordan's Site</title>
+    <link href="style.css" rel="stylesheet">
+    <script src="script.js"></script>
+</head>
 <body onload="generateNavbar() ; getMessages()">
-    <div id="content">
+<div id="content">
     <div class="navbar" id="jordanNav">
     </div>
     <div id="content-center">
-      <h1> Comments </h1>
-      <h3> Leave a Comment Below </h3>
+        <<<<<<< Updated upstream
```

Comment:
Remove? Also make sure you got the merge results correct

Reply:
Addressed